tourneydefs.py

```diff
@@ -456,9 +460,12 @@ def get_standings(self):
                 standing_data[team.id] += int(team.points)
 
         for match in self.matches.values():
-            if match.winner != 2:
+            if match.winner not in [2, 3]:
                 winner = match.teams[match.winner]
                 standing_data[winner] += 1
+            elif match.winner == 3:  # Tie
```

Comment:
This is an int up until now, could anything go sideways if it becomes a float? It feels like Bo2 world should probably be 2 points win 1 point each draw

Reply:
I checked the txt outputs after a tie, and it all seemed to work as expected. Further matches added/finished after a Bo2 tie also appeared to work as expected.